run.sh

```diff
@@ -30,6 +32,7 @@ run_cmd() {
         sleep 0.01
         ctr=$((ctr+1))
     done
+    sleep 1
```

Comment:
All these changes are already in master: https://github.com/aravindhank11/llama2-expr/blob/master/run.sh
Can you please rebase. Something seemse wrong